### ISSUE TITLE
[4.0.0-beta.6] createClassProxy TypeError with styled components (react-emotion)

### ISSUE THREAD

**Issue by apostolos** [gaearon/react-hot-loader#745]
### Description

Whenever a styled component is mounted an exception is thrown.

### Actual behavior

```
createClassProxy.js:83 Uncaught TypeError: Cannot assign to read only property 'toString' of function 'class extends InitialParent {
    constructor(props, context) {
      super(props, context);
      this[_symbol...<omitted>... }'
    at proxyClass (createClassProxy.js:83)
    at exports.updateProxyById (proxies.js:49)
    at exports.createProxyForType.type (proxies.js:56)
    at resolveType (patch.dev.js:58)
    at Object.patchedCreateElement [as createElement] (patch.dev.js:75)
    at ContactNameRoute (index.jsx:134)
    at ProxyComponent.render (createClassProxy.js:70)
    at finishClassComponent (react-dom.development.js:7873)
    at updateClassComponent (react-dom.development.js:7850)
    at beginWork (react-dom.development.js:8225)
```

### Environment

React Hot Loader version: 4.0.0-beta.6
React Version: 16.2.0
Emotion: 9.0.0-1
React-emotion: 9.0.0-1

### Reproducible Demo

```js
import * as React from 'react';
import styled from 'react-emotion';
import { hot } from 'react-hot-loader';

const TestDiv = styled('div')`
  background: red;
`;

// doesn't work
const App = () => <TestDiv>Test</TestDiv>;

// works
//const App = () => <div>Test</div>;

export default hot(module)(App);
```

**Comment by neoziro** [comment]
We have another problem #746. I will fix the `toString` problem.

**Comment by neoziro** [comment]
Fixed in #746

**Comment by neoziro** [comment]
Fixed in [v4.0.0-beta.7](https://github.com/gaearon/react-hot-loader/releases/tag/v4.0.0-beta.7)
